docs/aurora/debugging/debugging-overview.md

```diff
@@ -4,4 +4,8 @@ There are 3 debuggers available on Aurora:
 
 1. [gdb-oneapi](./gdb-oneapi.md) - This is Intel's version of gdb augmented to allow debugging kernels executing on the PVC GPUs.
 2. [DDT](./ddt-aurora.md) - The Linaro parallel debugger. This is the same parallel debugger that we have on Polaris. It supports a client-server mode and (via using `gdb-oneapi` internally) debugging kernels executing on the PVC GPUs.
-3. [gdb4hpc](./gdb4hpc.md) - An alternative for CPU debugging only that will apply commands to all threads in the MPI process group.
\ No newline at end of file
+3. [gdb4hpc](./gdb4hpc.md) - An alternative for CPU debugging only that will apply commands to all threads in the MPI process group.
+
+There is also a correctness tool for Fortran:
+
+1. [Codee](./codee.md)
```

Comment:
add a short description after the page link, like above

Reply:
doesnt it also support C, C++? Was reading https://docs.nersc.gov/tools/performance/codee/ --- are there plans to add a short walkthrough like they do at some point?

Reply:
We can add a walkthrough at some point, but right now I just wanted to get a bare minimum page together and on the website. The Codee website has a lot of "quickstarts" already too.

Reply:
thanks for the careful review, it's appreciated!